keycloak-389-umbrella/values.yaml

```diff
@@ -18,7 +18,7 @@ keycloak:
   persistence:
     enabled: false
   directoryManagerPassword: changeme
-  initCommand: sleep 10 && (/usr/sbin/dsconf localhost suffix get "dc=opennetworking,dc=org" || /usr/sbin/dsconf localhost backend create --suffix 'dc=opennetworking,dc=org' --be-name=aether --create-entries --create-suffix)
+  initCommand: sleep 60 && /usr/sbin/dsconf localhost backend create --suffix 'dc=opennetworking,dc=org' --be-name=aether --create-entries --create-suffix
```

Comment:
Hmm - the idea of the `or` is so that **if** this gets run a 2nd time it will not give the error `already exists`.
K8S might run the postStart hook more than once

Does it work if you **only** change the `sleep` duration

Reply:
for me the suffix command didn't work it gave me no such option error (is that command from any image tag?)
And by just incrementing the sleep and using the create command after it works on install